platforms/access-bridge/test/unit/logger.test.ts

```diff
@@ -4,9 +4,26 @@ import * as Sentry from '@sentry/node';
 import logger from '../../src/pipeline/logger.js';
 
 describe('Logger Tests', () => {
+  // Preserve the original console methods
+  const originalConsole = { ...console };
```

Comment:
If you use a logger service this one may help:

```
  mockService(
    LogTransporter,
    __debug__
      ? new ConsoleTransporter(LogLevel.DEBUG)
      : {
          log() {},
        },
  );

```

Reply:
It is possible that the memory issue we are facing is connected because the logger was outputting error objects.